scripts/targets-common.R

```diff
@@ -40,19 +28,6 @@ if (debug_mode) {
 }
 
 tar_option_set(
-  packages = c(
-    "assertthat",
-    "dplyr",
-    "epieval",
-    "epipredict",
-    "ggplot2",
-    "lubridate",
-    "parsnip",
-    "tibble",
-    "tidyr",
-    "epidatr"
-  ), # packages that your targets need to run
```

Comment:
pretty certain we need to include the packages to get the targets to run in some way shape or form. iirc including them here is faster.

Reply:
gotcha, yea [this section](https://books.ropensci.org/targets/packages.html#loading-and-configuring-r-packages) says it's faster because tar_option_set lazy loads packages. i was just confused about the many places imports were being handled, so i centralized them in `imports.R` and didn't notice a big performance hit. i can try moving some back over here and see if it's faster. we don't really use `tar_visnetwork()` anyway, but maybe `tar_manifest()` is affected.

Reply:
I would mostly worry about what happens in parallelization actually, but if those run fine than it's fine

Reply:
I guess we've confirmed this works in parallel w/out these?

Reply:
yup!